src/OneDriveSdk/Microsoft.OneDrive.Sdk.csproj

```diff
@@ -67,17 +67,17 @@
     <Compile Include="Requests\UploadSessionRequest.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.Graph.Core, Version=1.1.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Microsoft.Graph.Core.1.1.1\lib\portable45-net45+win8+wpa81\Microsoft.Graph.Core.dll</HintPath>
+    <Reference Include="Microsoft.Graph.Core, Version=1.2.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
Would you mind merge the updated master branch into your change? I think we have already upgraded to consumer the Graph.Core 1.2.1